lib/src/package_analyzer.dart

```diff
@@ -39,6 +39,10 @@ class InspectOptions {
   /// Whether pana should also access the remote repository specified in pubspec.yaml.
   final bool checkRemoteRepository;
 
+  /// If specified, pana will analyze the package with future SDKs configured in
+  /// [ToolEnvironment], and grant this tag if the analysis passes without errors.
+  final String? futureSdkTag;
```

Comment:
This is a path to a future SDK, right?
```suggestion
  final String? futureSdk;
```